boards/arm/phy62xx/phy6222/src/reset.c

```diff
@@ -0,0 +1,59 @@
+/****************************************************************************
+ * boards/arm/stm32/stm32f103-minimum/src/stm32_reset.c
```

Comment:
?

Reply:
will fix next update..

Reply:
you must fix all issues in this patch before it could get approved.